### PR TITLE
Fixed potion command

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@Alexander01998 hey!

**Comment by Alexander01998** [comment]
The front matter and the syntax need to be updated before I can merge this. See:
- https://github.com/Wurst-Imperium/wurst-client.tk/blob/gh-pages/wiki/Commands/addalt/index.md
  for the front matter
- https://github.com/Wurst-Imperium/Wurst-Client/blob/master/Wurst%20Client/src/tk/wurst_client/commands/PotionCmd.java
  for the syntax.

**Comment by ghost** [comment]
i didn't understand
https://github.com/Wurst-Imperium/wurst-client.tk/blob/gh-pages/wiki/Commands/addalt/index.md

**Comment by Alexander01998** [comment]
It now uses `category` instead of `tags`.

**Comment by ghost** [comment]
Fixed @Alexander01998

File: wiki/Commands/potion/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+category: Command
+---
+Adds, sets or removes effects of the held potion. Accepts arguments in groups of 3 for add and set modes. Can set multiple effects to one potion. Make sure it fits the chat bar.
+
+## Syntax
+
+- `remove <effect>`
+- `add (<effect> <amplifier> <duration>)...`
+- `set (<effect> <amplifier> <duration>)...`
+or longer:
```
Comment (Alexander01998) [inline wiki/Commands/potion/index.md]: The "or longer" part doesn't make much sense anymore, as the syntax above it already covers the longer versions.

File: wiki/Commands/potion/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- `.potion <add/set> <id/name> <amplifier> <duration> <id/name> <amplifier> <duration>`
+
+- `add` adds effects
+- `set` removes current effect and changes them with given effects
+- `remove` removes effects
+
+## Example
+`.potion set 14 0 60 5 24 75` would set the held potion's effects to invisibility for 60 seconds and strength 25 for 75 seconds.
```
Comment (Alexander01998) [inline wiki/Commands/potion/index.md]: This example should use the effect names to make it easier to understand.

File: wiki/Commands/potion/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+![.potion](https://cloud.githubusercontent.com/assets/11584045/8901300/74b61982-3450-11e5-9f3a-0a69402c380c.jpg)
```
Comment (Alexander01998) [inline wiki/Commands/potion/index.md:22]: This image should have a more descriptive `alt` text.

**Comment by Alexander01998** [comment]
Added some further notes.

**Comment by ghost** [comment]
Fixed. (I guess..)

**Comment by Alexander01998** [comment]
Not quite. I'll take care of that.

**Pull request merged**